### ISSUE TITLE
FSE rollout: remove the FSE option toggle and screen

### ISSUE THREAD

**Issue by vindl** [Automattic/wp-calypso#60256]
### Details

This amounts to rolling out FSE for all existing sites that have Universal Theme active. We should do this only after both `/new` and `/start` flows are released to 100% of users.

### Checklist

- [x] Remove FSE option toggle and screen from Atomic sites
- [x] Remove FSE option toggle and screen from Simple sites (proxied only)
- [x] Activate FSE by default for all sites on Universal themes

**Comment by vindl** [comment]
Based on our Slack convo: Let's put this task on hold until other rollout tasks are finished.
